src/server/symbols.map

```diff
@@ -1,4 +1,4 @@
-MIR_SERVER_2.15 {
+MIR_SERVER_2.16 {
```

Comment:
`MIR_SERVER_2.17` - 2.16 has already been released

Reply:
Cool! Why did it say 2.15? Because we didn't introduce any new symbols on the prev release?

Reply:
Exactly - the stanza is the release the symbol was introduce to the current .soname